install/install.sh

```diff
@@ -93,10 +101,28 @@ echo "Checking for docker."
 ## We unset this variable for this command to avoid conflicts with companion version
 docker --version || curl -fsSL https://get.docker.com | env -u VERSION sh || (
     echo "Failed to start docker, something is wrong."
-    echo "Please report this problem."
-    exit 1
+    echo "Are in in CI? trying dind."
 )
 systemctl enable docker
+if [ $RUNNING_IN_CI -eq 1 ]
+then
+
+
+set -x && \
+    addgroup --system dockremap && \
+    adduser --system --ingroup dockremap dockremap && \
+    echo 'dockremap:165536:65536' >> /etc/subuid && \
+    echo 'dockremap:165536:65536' >> /etc/subgid
+
+    dind dockerd $DOCKER_EXTRA_OPTS &
+      while(! docker info > /dev/null 2>&1); do
+        echo "==> Waiting for the Docker daemon to come online..."
+        sleep 1
+      done
+    alias docker=dind
+fi
+
+sudo usermod -aG docker pi
```

Comment:
Do we want to give normal user docker access ?

Reply:
good question. I'll try to run it whitout this line and see what happens